os_net_config/__init__.py

```diff
@@ -128,34 +188,69 @@ def add_route_table(self, route_table):
         """
         raise NotImplementedError("add_route_table is not implemented.")
 
+    def del_route_table(self, route_table):
+        """Delete a route table object to the net config object.
+
+        :param route_table: The RouteTable object to be deleted.
+        """
+        raise NotImplementedError("del_route_table is not implemented.")
+
     def add_interface(self, interface):
         """Add an Interface object to the net config object.
 
         :param interface: The Interface object to add.
         """
         raise NotImplementedError("add_interface is not implemented.")
 
+    def del_interface(self, interface):
+        """Delete the Interface object to the net config object.
+
+        :param interface: The Interface object to be deleted.
+        """
+        raise NotImplementedError("del_interface is not implemented.")
+
     def add_vlan(self, vlan):
         """Add a Vlan object to the net config object.
 
         :param vlan: The vlan object to add.
         """
         raise NotImplementedError("add_vlan is not implemented.")
 
+    def del_vlan(self, vlan):
+        """Delete a Vlan object to the net config object.
+
+        :param vlan: The vlan object to be deleted.
+        """
+        raise NotImplementedError("del_vlan is not implemented.")
+
     def add_bridge(self, bridge):
         """Add an OvsBridge object to the net config object.
 
         :param bridge: The OvsBridge object to add.
         """
         raise NotImplementedError("add_bridge is not implemented.")
 
+    def del_bridge(self, bridge):
+        """Delete an OvsBridge object to the net config object.
+
+        :param bridge: The OvsBridge object to be deleted.
+        """
+        raise NotImplementedError("del_bridge is not implemented.")
+
     def add_ovs_user_bridge(self, bridge):
         """Add an OvsUserBridge object to the net config object.
 
         :param bridge: The OvsUserBridge object to add.
         """
         raise NotImplementedError("add_ovs_user_bridge is not implemented.")
 
+    def del_ovs_user_bridge(self, bridge):
+        """Delete an OvsUserBridge object to the net config object.
+
+        :param bridge: The OvsUserBridge object to be deleted.
+        """
+        raise NotImplementedError("del_ovs_user_bridge is not implemented.")
+
```

Comment:
Is there no valid use case for del_linux_bridge () ? Controller nodes might have created linux_bridges